client/ListingEntry.jsx

```diff
@@ -1,40 +1,44 @@
-import React, { Component } from 'react';
-import _ from 'underscore';
+import React from 'react';
+import propTypes from 'prop-types';
+import ReactStars from 'react-stars';
 import style from './entrystyle.css';
 
-class ListingEntry extends Component {
-  constructor(props) {
-    super(props)
-    this.rating = Math.round(this.props.listing.reviews.reduce((accumulator, current) => {
```

Comment:
May want to declare `const reviews = this.props.listing.reviews`, to improve readability here.

Reply:
Never mind, I didn't realize you had already deleted this!